K8s/api-deployment.yaml

```diff
@@ -69,7 +69,7 @@ spec:
               value: ctrl
             - name: TRANSFER_SPEED_MIN
               value: "10"
-          image: sodafoundation/multi-cloud-api:latest
+          image: sodafoundation/multi-cloud-api:RELEASE_VERSION
```

Comment:
RELEASE_VERSION :This is replaced by the ansible installer and set to the right version being installed. Hardcoding 'latest' will not be a good idea, as it may not match with the released multi-cloud codd base. Having it configurable makes multi-cloud-HA deployment compatible with the multi-cloud code & release.

Reply:
Ok but who sets this RELEASE_VERSION? Ansible installer ? Can the multicloud HA be installed without using installer? Independent multicloud installation

Reply:
Just was browsing Installer code, RELEASE_VERSION is not mentioned